R/inflate.R

```diff
@@ -456,19 +466,21 @@ create_vignette <- function(parsed_tbl, pkg, vignette_name, open_vignette = TRUE
   head <- create_vignette_head(pkg = pkg, vignette_name = vignette_name)
 
   # Write vignette
-  if (nrow(vignette_tbl) == 0) {
-    lines <- c(head,
-               "",
-               "<!-- This vignette is generated by fusen: do not edit by hand -->\n"
+  lines <- c(
+    head,
+    "",
+    sprintf(
+      "<!-- This vignette is generated by {fusen} from %s: DO NOT EDIT BY HAND -->\n",
+      relative_flat_file
     )
-  } else {
+  )
+  if ( nrow(vignette_tbl) != 0 ) {
```

Comment:
Here I removed some duplicated code by inverting the logical test in the `if` statement.